src/app/(main)/connections/_components/connection-table.tsx

```diff
@@ -19,8 +19,8 @@ import {
   TableHeader,
   TableRow,
 } from "@scylla-studio/components/ui/table";
-import type { Connection } from "@scylla-studio/lib/internal-db/connections";
-import { useEffect, useState, useTransition } from "react";
+import type {Connection} from "@scylla-studio/lib/internal-db/connections";
+import {useEffect, useState, useTransition} from "react";
```

Comment:
Your editor be removing spaces [😭](https://emojipedia.org/loudly-crying-face)

Reply:
i'm sorry man i'll tell him to not remove next time ok?